docs/guides/release/linux.md

```diff
@@ -0,0 +1,33 @@
+# Linux Releasing
+
+- Set up your [build environment](/docs/guides/build/linux.md:3).
+
+## Github Releases
+
+- Install [github-release](https://github.com/github-release/github-release).
+- [Generate a personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token)
+  and set it to the environment variable `GITHUB_TOKEN`.
+
+### Debian Package
+- Install a series of build tools.
+Debuild
+Deb Helper (`dh`)
+Equivis (`equivis-build`)
+- Run [release_linux_desktop_debian.sh](/utils/dev/build-lockbook-debian/release_linux_desktop_debian.sh).
```

Comment:
You want to continue maintaining debs now that we have snaps?

Reply:
I'm fine with it either way

Reply:
I don't want to continue maintaining debs, especially because they wouldn't work on some debian distributions and versions. I feel snap is better, and in the future flatpak.